test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -317,7 +317,7 @@
     <Reference Include="System.Net.Http.WebRequest" />
     <Reference Include="System.Numerics" />
     <Reference Include="System.Reflection.Metadata, Version=1.4.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\System.Reflection.Metadata.1.4.1\lib\portable-net45+win8\System.Reflection.Metadata.dll</HintPath>
+      <HintPath>..\..\packages\System.Reflection.Metadata.1.4.1-beta-24227-04\lib\portable-net45+win8\System.Reflection.Metadata.dll</HintPath>
```

Comment:
Yeah, sorry -- in addition to moving WebJobs nugets forward, I consolidated the 12 packages that had issues. Most were in the Integration tests. 

I went ahead and moved this to 1.4.1.